crates/aptos-rest-client/src/state.rs

```diff
@@ -49,7 +49,15 @@ impl State {
                 oldest_ledger_version,
             }
         } else {
-            todo!()
+            anyhow::bail!(
```

Comment:
Might have to handle if the version doesn't have oldest_ledger_version, since the branch cut for testnet I don't think has it.  You could probably make it optional.

Reply:
I've been cherry picking changes to deal with that for now. But yeah it'd be nice to exercise a similar practice to RPC spec evolution.